PHP/CodingConventions.md

```diff
@@ -24,11 +26,66 @@ found in the [MediaWiki Coding Conventions](http://www.mediawiki.org/wiki/Manual
 
 Functions should have a limited length.  This limit is loosely defined as one “page”, where a page is the number
 of lines that fit into your editor window or less.  This definition is intentionally vague to allow flexibility but the
-spirit of this guideline is that a developer should not have to scroll to see a full function definition.
+spirit of this guideline is that a developer should not have to scroll vertically to see a full function definition.
 
 If a single function becomes too long it should be broken up into smaller functions which the original function can
 call.
 
+### Line Length
+
+Lines should be limited to 120 characters.  This limit is chosen as a typical IDE default for line breaks.  The spirit
+of this guideline is that a developer should not have to scroll horizontally to see the end of a line.
+
+If a single line becomes too long, newlines should be added at appropriate breakpoints.
+
+### Conditional Logic
+
+Conditional tests should not do more than two checks on a single line.  If a conditional requires more than two checks,
+one of the following remedies should be used:
+ 
+* each check should be split into it's own line
+* the single conditional should be split into separate conditionals of two or less checks
+* all checks replaced by a function call
+
+Example:
+
+```php
+// Bad
+if ( F::App()->wg->User->can('edit') && F::App()->wg->Skin->getSkinName() == 'oasis' && empty( F::app->wg->NoExternals ) {
```

Comment:
We should make sure these examples follow our white space guidelines as well. ```F::App()->wg->User->can('edit')``` should be ```F::App()->wg->User->can( 'edit' )```